src/main/java/seedu/duke/Ui.java

```diff
@@ -0,0 +1,159 @@
+package seedu.duke;
+
+import java.util.Scanner;
+
+/**
+ * Ui object class.
+ */
+public class Ui {
+    /** Scanner to take user input */
+    private final Scanner sc;
+
+    /**
+     * Creates a Ui object.
+     */
+    public Ui() {
+        this.sc = new Scanner(System.in);
+    }
+
+    /**
+     * Returns text to display on bot initialization.
+     *
+     * @return String for welcome.
+     */
+    public String spawnBot() {
+        return "HEY! I'm GRUMMY!\nHow can I help you?";
+    }
+
+    /**
+     * Returns text to display on bot exit.
+     *
+     * @return String for exit.
+     */
+    public String displayExit() {
+        return "Goodbye! Hope to see you again :>";
+    }
+
+    /**
+     * Returns the number of tasks in the TaskList.
+     *
+     * @param taskList the TaskList to return the number of tasks from.
+     *
+     * @return String for tasks left.
+     */
+    public String displayTaskList(TaskList taskList) {
+        if (taskList.size() == 0) {
+            return "You have no tasks. Hooray!";
+        } else {
+            int index = 1;
+            String list = "";
+            while (index < (taskList.size() + 1)) {
+                list = list + "\n   " + index + ". " + taskList.get(index - 1).toString();
+                index++;
+            }
+            String hasTasks = "Here are your tasks:";
```

Comment:
Would it be cleaner to return in a single statement without the hasTasks String?

Reply:
Amended!